ionoscloud/data_source_apigateway_route.go

```diff
@@ -11,24 +11,26 @@ import (
 	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils"
 )
 
-func dataSourceApiGatewayRoute() *schema.Resource {
+func dataSourceAPIGatewayRoute() *schema.Resource {
 	return &schema.Resource{
-		ReadContext: dataSourceApiGatewayRouteRead,
+		ReadContext: dataSourceAPIGatewayRouteRead,
 		Schema: map[string]*schema.Schema{
 			"id": {
 				Type:        schema.TypeString,
-				Description: "The ID (UUID) of the API Gateway route.",
+				Description: "The ID (UUID) of the API Gateway Route.",
 				Optional:    true,
+				Computed:    true,
 			},
-			"apigateway_id": {
+			"gateway_id": {
```

Comment:
Just a note that I agree with this change and I'll do this for the Gateway resource too